src/docgen.js

```diff
@@ -29,10 +29,11 @@ function configurePackage(p) {
      // build navigation
      .processor(require('./processors/structuredParam'))
 
+     /* commenting out the following because it's not a default package
      // add the JSX file reading service to the processor
      .config(function (readFilesProcessor, jsxFileReader) {
         readFilesProcessor.fileReaders.unshift(jsxFileReader);
-     })
```

Comment:
entire block needs to be removed, it looks like obsolete code, it's already exists in jsx package